docs/contribute/plugin/develop_plugin_c.md

```diff
@@ -4,199 +4,238 @@ sidebar_position: 2
 
 # Develop WasmEdge Plug-in in C API
 
-WasmEdge provides a C++ based API for registering extension modules and host functions. While the WasmEdge language SDKs allow registering host functions from a host (wrapping) application, the plugin API allows such extensions to be incorporated into WasmEdge's own building and releasing process.
-
-## Prerequisites
-
-For developing the WasmEdge plug-in in C API, please [install WasmEdge](../../start/install.md#install) first.
+By developing a plug-in, one can extend the functionality of WasmEdge and customize it to suit specific needs. WasmEdge provides a C++ based API for registering extension modules and host functions. While the WasmEdge language SDKs allow registering host functions from a host (wrapping) application, the plug-in API allows such extensions to be incorporated into WasmEdge's building and releasing process. Here is a flowchart showing all the steps needed for developing WasmEdge Plug-in -
```

Comment:
"WasmEdge provides C based APIs for ... "

Reply:
Jut to double check, in previous doc it was written as - 
> [WasmEdge provides a C++ based API for registering extension modules and host functions.](https://wasmedge.org/docs/contribute/plugin/develop_plugin_c)

So will it be c or c++ plugin?

Reply:
It's C API. The previous version is wrong.

Reply:
Thank you for your clarification @alabulei1.